operators/src/plot/histogram.rs

```diff
@@ -983,7 +984,9 @@ mod tests {
 
         let mut execution_context = MockExecutionContext::default();
         execution_context.add_meta_data(
-            DatasetId::Internal(dataset_id),
+            DatasetId::Internal {
+                dataset: dataset_id,
```

Comment:
```suggestion
                id: dataset_id,
```

Reply:
I think it's more consistent to call the dataset_id `dataset` in the internal and external case